docs/GettingStartedOnLinux.md

```diff
@@ -0,0 +1,211 @@
+
+You can check by typing `which npm` into the terminal.
+
+If have NPM, you'll see something like:
+
+```bash
+$ /usr/local/bin/npm
+```
+
+If you don't have NPM, you'll see:
+
+```bash
+$ which npm
+npm not found
+$
+```
+
+If that's the case, run the following command to install NPM:
+
+`curl http://npmjs.org/install.sh | sh`
+
+#### Installing watchman
+
+[watchman](https://facebook.github.io/watchman/docs/install.html) is a tool developed by Facebook for watching the filesystem. For best results, you'll want to install it.
+
+Once again, paste these into your terminal to get up and running:
+
+* `git clone https://github.com/facebook/watchman.git`
+* `cd watchman`
+* `git checkout v4.1.0  # the latest stable release`
+* `./autogen.sh`
+* `./configure`
+* `make`
+* `sudo make install`
+
+#### Installing flow
+
+Next up: flow. This is a static type checker for Javascript.
+
+Here's how you install it:
+
+* `wget https://facebook.github.io/flow/downloads/flow-linux64-latest.zip`
+* `unzip flow-linux64-latest.zip`
+* `cd flow`
+* `sudo mv flow /usr/local/bin/flow`
+
+Note: for advanced users, you can put the `flow` executable anywhere on your PATH. It doesn't have to be in `/usr/local/bin`.
+
+#### Installing React Native
+
+OK, with all of that out of the way, it's finally time to install React Native.
+
+For most users of this guide, you'll need to use `sudo` to install React Native as root:
+
+* `sudo npm install -g react-native-cli`
+
+For users who installed NPM through other means, you'll likely want to install React Native as a normal user:
+
+* `npm install -g react-native-cli`
+
+## Create Project
+
+Hooray! React Native is installed.
+
+Now it's time to do what we came here to do: make an app!
+
+To create an app project, you'll need to sit down and input this command:
+
+* `react-native init AwesomeProject`
+
+It'll take several minutes, so don't panic if your terminal sits around for a while before spitting out any output.
+
+## Setting Up an Android Device
+
+OK, now that we have a starter project waiting for us, we're almost ready to run the code.
+
+But before we do, we'll need either an emulator or a real Android device to run our code on.
+
+In this tutorial, we're going to use a real Android device.
+
+First thing is to plug in your device and check the manufacturer code by using `lsusb`:
+
+```bash
+$ lsusb
+Bus 002 Device 002: ID 8087:0024 Intel Corp. Integrated Rate Matching Hub
+Bus 002 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+Bus 001 Device 003: ID 22b8:2e76 Motorola PCS 
+Bus 001 Device 002: ID 8087:0024 Intel Corp. Integrated Rate Matching Hub
+Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+Bus 004 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
+Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+```
+
+Seems like a bunch of gibberish, eh? You want the line that represents your phone. If you're in doubt, try unplugging your phone and running the command again:
```

Comment:
Instead of `Seems like a bunch of gibberish, eh?`, we might write `These lines represent the USB devices currently connected to your machine.`